import/textselection/TextSelectionHandle.qml

```diff
@@ -0,0 +1,140 @@
+/****************************************************************************
+**
+** Copyright (C) 2013-2016 Jolla Ltd.
```

Comment:
Please change to lower-case (c) and "2013 - 2020 Jolla Ltd." and add:

** Copyright (c) 2020 Open Mobile Platform LLC.

And remove contacts